src/Robo/Blt.php

```diff
@@ -88,6 +93,18 @@ public function __construct(
     $this->runner->setRelativePluginNamespace('Blt\Plugin');
 
     $this->setLogger($container->get('logger'));
+
+    $this->initializeAmplitude();
+  }
+
+  /**
+   * Initializes Amplitude.
+   */
+  private function initializeAmplitude() {
+    $amplitude = Amplitude::getInstance();
+    $amplitude->init('dfd3cba7fa72065cde9edc2ca22d0f37')
```

Comment:
@alexxed I don't seem to have sufficient access to Amplitude to view API keys, so I have no idea to which project this is reporting. Do you know which project is associated with this API key?

Reply:
@raghunat Alex said you might know where this API key came from, so I can check whether this is reporting correctly?